src/platforms/esp32/components/avm_builtins/CMakeLists.txt

```diff
@@ -55,7 +56,7 @@ endif()
 idf_component_register(
     SRCS ${AVM_BUILTIN_COMPONENT_SRCS}
     INCLUDE_DIRS "include"
-    PRIV_REQUIRES "libatomvm" "avm_sys" "nvs_flash" "driver" "esp_event" "esp_wifi" ${ADDITIONAL_PRIV_REQUIRES}
+    PRIV_REQUIRES "libatomvm" "avm_sys" "nvs_flash" "driver" "esp_event" "esp_wifi" "fatfs" ${ADDITIONAL_PRIV_REQUIRES}
```

Comment:
should this only be if AVM_ENABLE_STORAGE_NIFS

eg remove "fatfs" here and add this above?

```
if(AVM_ENABLE_STORAGE_NIFS)
   set(ADDITIONAL_PRIV_REQUIRES ${ADDITIONAL_PRIV_REQUIRES} "fatfs")
endif()
```

Reply:
Unused functions are pruned, so this is not a problem.